indico/modules/events/operations.py

```diff
@@ -235,6 +235,7 @@ def _log_event_update(event, changes, extra_log_fields, visible_person_link_chan
         'address': 'Address',
         'venue_room': {'title': 'Location', 'type': 'string'},
         'keywords': 'Keywords',
+        'render_mode': {'title': 'Render Mode', 'type': 'number'},
```

Comment:
That's not a number but an enum, and usually enums are correctly auto-detected...

Reply:
Fixed: I added it as a number as HiddenField had numerical values, but I see it's automatically detected as enum, and is working well :)

<img width="813" alt="image" src="https://github.com/user-attachments/assets/6cd5500f-14e7-43f1-bf78-11d72c25ba79">